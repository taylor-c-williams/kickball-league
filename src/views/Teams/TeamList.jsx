import React from 'react';

export default function TeamList() {
	return <div></div>;
}
