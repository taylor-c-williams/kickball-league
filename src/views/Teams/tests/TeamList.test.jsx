import { render, screen } from '@testing-library/react';
import TeamList from '../TeamList';

it('renders TeamList elements', () => {
	const { container } = render(<TeamList />);

	const textElement = screen.getByText('Teams', { exact: false });
	expect(textElement).toBeInTheDocument();
	expect(container).toMatchSnapshot();
});
