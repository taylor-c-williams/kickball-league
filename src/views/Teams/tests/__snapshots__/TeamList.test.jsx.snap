// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TeamList elements 1`] = `
<div>
  <div
    class="content"
  >
    <h1>
      Teams:
    </h1>
    <ul />
    <a
      href="/create"
    >
      Create New Team
    </a>
  </div>
</div>
`;
